designer/client/src/reducers/graph/utils.fixtures.ts

```diff
@@ -167,7 +167,9 @@ export const state: GraphState = {
                 name: "NOT_DEPLOYED",
             },
             version: null,
+            applicableActions: ["DEPLOY", "ARCHIVE", "RENAME"],
```

Comment:
It is not easy to understand what is the difference between applicable vs allowed actions. Maybe we should refactor the original naming? How they different? One status make action buttons visible and another make them clickable (enabled)?

Reply:
I renamed applicableActions to `visibleActions` and left the `allowedActions` unchanged. That way we have lists of `visible` and `allowed` actions, IMO that is clearer.